iptables/iptables.go

```diff
@@ -173,7 +175,7 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	args = append([]string{ipt.path}, args...)
 	if ipt.hasWait {
```

Comment:
It seems that the test are failing because of the condition (see link below) "forcing" `hasWait` to `false` when it is `true`.
See https://github.com/coreos/go-iptables/blob/master/iptables/iptables_test.go#L82-L88.